### PR TITLE
Remove duplicated icons

### PR DESCRIPTION
**Pull request by kwwzzz**
Remove all these duplicated icons:

- devices/printer.png
- emblems/config-users.png
- mimetypes/application-x-addon.png
- categories/org.gnome.Settings-multitasking-symbolic.svg
- mimetypes/x-office-address-book-symbolic.svg
- ui/adw-expander-arrow-symbolic.svg
- ui/hdy-expander-arrow-symbolic.svg

Closes https://github.com/ubuntu/yaru/issues/2557
Although this type of problem has been dealt with before, there are still some duplicate icons.

### CODE REVIEW

File: icons/src/symlinks/symbolic/categories.list [hunk 1]
```diff
@@ -3,7 +3,6 @@ applications-utilities-symbolic.svg applications-accessories-symbolic.svg
 calendar-today-symbolic.svg view-tasks-today-symbolic.svg
 calendar-week-symbolic.svg view-tasks-week-symbolic.svg
 diagnostics-symbolic.svg org.gnome.Settings-diagnostics-symbolic.svg
-multitasking-symbolic.svg org.gnome.Settings-multitasking-symbolic.svg
 multitasking-symbolic.svg preferences-desktop-multitasking-symbolic.svg
```
Comment (3v1n0) [inline icons/src/symlinks/symbolic/categories.list:6]: Shouldn't we keep `org.gnome.Settings-multitasking-symbolic.svg` instead?

Comment (kwwzzz) [reply icons/src/symlinks/symbolic/categories.list:6]: In fact this article is already [here](https://github.com/ubuntu/yaru/blob/master/icons/src/symlinks/symbolic/apps.list#L97)， so it is still duplicated

**Comment by Jupi007** [comment]
Thanks a lot @kwwzzz !

@3v1n0 Please wait before merging this PR, I'd like to check if the correct icons has been removed.
Also, I think we should create a script which would check for duplicated icons.

**Comment by 3v1n0** [comment]
CI jobs errors should be clearer now:

```
 icons/meson.build:287:10: ERROR: Problem encountered: Symbolic links are pointing to invalid paths:
  Yaru/16x16@2x/mimetypes/extension.png
  Yaru/16x16@2x/mimetypes/application-x-xpinstall.png
  Yaru/48x48/mimetypes/extension.png
  Yaru/48x48/mimetypes/application-x-xpinstall.png
  Yaru/256x256@2x/mimetypes/extension.png
  Yaru/256x256@2x/mimetypes/application-x-xpinstall.png
  Yaru/24x24/mimetypes/extension.png
  Yaru/24x24/mimetypes/application-x-xpinstall.png
  Yaru/48x48@2x/mimetypes/extension.png
  Yaru/48x48@2x/mimetypes/application-x-xpinstall.png
  Yaru/16x16/mimetypes/extension.png
  Yaru/16x16/mimetypes/application-x-xpinstall.png
  Yaru/32x32/mimetypes/extension.png
  Yaru/32x32/mimetypes/application-x-xpinstall.png
  Yaru/256x256/mimetypes/extension.png
  Yaru/256x256/mimetypes/application-x-xpinstall.png
  Yaru/24x24@2x/mimetypes/extension.png
  Yaru/24x24@2x/mimetypes/application-x-xpinstall.png
  Yaru/32x32@2x/mimetypes/extension.png
  Yaru/32x32@2x/mimetypes/application-x-xpinstall.png
```

**Review by Jupi007 — CHANGES_REQUESTED** [review]
Sorry for the late reply.
I have a few requested changes :)

- when you remove a symlink, you should also remove its entry inside `/icons/src/symlinks` ;
- the `printer` icon should be located in the `/devices` folder, like the duplicated icon you removed, but using a symlink. So in **devices.list** add a new entry `../actions/document-print.png printer.png`, and remove the current one in **actions.list** ;
- about both the `*-expander-arrow-symbolic` icons, you removed the wrong symlink. It should be symlinked to `pan-down-symbolic` instead of `do-down-symbolic`.

For now, that's all I noticed.

**Comment by Muqtxdir** [comment]
Does having them cause any issue? it'd be nice if we could look into merging this after lunar if it's not that critical

**Comment by Jupi007** [comment]
I fully agree with you @Muqtxdir :+1:

**Comment by kwwzzz** [comment]
> Sorry for the late reply. I have a few requested changes :)
> 
>     * when you remove a symlink, you should also remove its entry inside `/icons/src/symlinks` ;
> 
>     * the `printer` icon should be located in the `/devices` folder, like the duplicated icon you removed, but using a symlink. So in **devices.list** add a new entry `../actions/document-print.png printer.png`, and remove the current one in **actions.list** ;
> 
>     * about both the *-expander-arrow-symbolic`icons, you removed the wrong symlink. It should be symlinked to`pan-down-symbolic`instead of`do-down-symbolic`.
> 
> 
> For now, that's all I noticed.

Thank you very much for your reply, your reply helped me to understand yaru better

**Comment by Jupi007** [comment]
> I have made some changes based on your suggestions, but there are some altered changes regarding printer.png and application-x-addon.png. Because I see the svg format of both images in the src/fullcolor/default/ directory, I think the printer.png is more suitable for the devices directory and the application-x-addon.png is more suitable for the mimetypes directory.

@kwwzzz I fully agree with you. It is just that I didn't wanted to confuse you with complex instructions on how to achieve this :D
But if you are ready for it, you can have a go :+1:

**Comment by kwwzzz** [comment]
> > I have made some changes based on your suggestions, but there are some altered changes regarding printer.png and application-x-addon.png. Because I see the svg format of both images in the src/fullcolor/default/ directory, I think the printer.png is more suitable for the devices directory and the application-x-addon.png is more suitable for the mimetypes directory.
> 
> @kwwzzz I fully agree with you. It is just that I didn't wanted to confuse you with complex instructions on how to achieve this :D But if you are ready for it, you can have a go +1

Yes, I have submitted my new changes via the new pr, perhaps you can see if my changes are correct

**Review by Jupi007 — APPROVED** [review]
I rechecked this PR, and it's a LGFM :+1: 

@Muqtxdir can you double check, just in case?

**Pull request merged**